docs/_build/html/_sources/methods.rst.txt

```diff
@@ -0,0 +1,444 @@
+For a controller onto a maximal conductance, for instance, the first column is the controller state and the second is the maximal conductance.
+All other outputs are ``time series x XYZ`` where XYZ is the serialized list of those properties. For example, in a ``xolotl`` object with 2 compartments ``AB`` and ``BC``
+with two conductances each: ``NaV`` and ``Kd``, the intrinsic currents would return in the form of a ``time steps x 4`` matrix where the columns would be ordered:
+Kd current from AB, NaV current from AB, Kd current from BC, NaV current from BC.
+
+If ``integrate`` is passed one argument, it interprets this as a matrix of injected currents (in nA). For example ::
+
+  % applies 0.2 nA to each compartment
+  x.integrate(0.2)
+  % applies 0.2 nA to first compartment of three
+  x.integrate([0.2 0 0])
+  % applies a time-varying current to first compartment
+  % where time is a vector of length x.t_end / x.dt
+  x.integrate([0.2*sin(time); zeros(time); zeros(time)])
+
+If ``integrate`` is passed two arguments, the first is ignored, and the second is treated as a matrix of clamped voltage. ::
+
+  % fixes the voltage of all compartments at -50 mV
+  x.integrate([], -50])
+  % clamps the second compartment in a two-compartment network
+  x.integrate([], [NaN -50] )
+
+.. _manipulate:
+
+manipulate
+^^^^^^^^^^
+
+Opens the GUI to permit real-time visualization of changing network parameters.
+The first argument specifies for which network properties to generate sliders (default is all of them). This takes the form of a call to the find_ function.
+The second argument specifies which visualization function should be used (default is ``plot``). ::
+
+  % minimum usage
+  x.manipulate
+  % specify sliders only for maximal conductances of HH compartment
+  x.manipulate('HH*gbar')
+  % specify the myPlot function for visualization
+  x.manipulate('HH*gbar', @myPlot)
+
+.. _plot:
+
+plot
+^^^^
+
+Plots voltage and intracellular calcium traces for each compartment.
+The voltage traces are colored based on the dominant current. A current is dominant when
+it is the outward current with the greatest magnitude and dV/dt is negative;
+or it is the inward current with the greatest magnitude and dV/dt is positive. ::
+
+  x.plot
+
+.. hint::
+
+  This is a static method of ``xolotl``.
+
+.. _replicate:
+
+replicate
+^^^^^^^^^
+
+::
+
+  x.replicate('cpplabObject', N)
+
+Replicates a ``cpplab`` object in the tree ``N`` times with all children included.
+This is useful for generating many identical compartments. To generate 100 ``HH`` compartments,
+where ``HH`` is already specified ::
+
+  x.replicate('HH', 100)
+
+.. _reset:
+
+reset
+^^^^^
+
+Resets all state variables to their initial condition. When the ``xolotl`` object is flagged
```

Comment:
describe what variables are reset, and to what values.

Reply:
done